### PR TITLE
📗 [wip] - Ruby Language Guide Update

### PR DESCRIPTION
**Pull request by teesloane**
Updates the language guide to 2.1. 

**Blocked:**

Do not merge until [sample app](https://github.com/CircleCI-Public/circleci-demo-ruby-rails/blob/2.1-orbs-config/.circleci/config.yml) has been updated on master with 2.1 config.

### CODE REVIEW

File: jekyll/_cci2/language-ruby.md [hunk 1]
```diff
@@ -38,8 +38,159 @@ Database images for use as a secondary 'service' container are also available on
 
 ## Sample Configuration
 
+{:.tab.rubyconf.Config_2__1}
 {% raw %}
 
+```yaml
+version: 2.1 # Use CircleCI 2.1 
+orbs: # use orbs to help shorten and reduce repetition in our config.
+  ruby: circleci/ruby@0.1.2
```
Comment (ganezasan) [inline jekyll/_cci2/language-ruby.md:47]: We can use the latest version of this orb!

```suggestion
  ruby: circleci/ruby@0.2.1
```

File: jekyll/_cci2/language-ruby.md [hunk 2]
```diff
@@ -139,180 +289,219 @@ A good way to start using CircleCI is to build a project yourself. Here's how to
...
 This key is used to issue warnings about breaking changes.
 
 ```yaml
-version: 2
+version: 2.1
 ```
 
-Next, add a `jobs` key. Each job represents a phase in your Build-Test-Deploy process. The sample app only needs a `build` job, so all other options are nested under that key.
+The bulk of our config exists in creating *reusable configuration*. This means
+using orbs, commands, executors, and commands. Your configuration does not have
```
Comment (ganezasan) [inline jekyll/_cci2/language-ruby.md:302]: There are two commands,  does it mean `Yaml anchors and aliases`?

```suggestion
using orbs, commands, executors, and Yaml anchors and aliases. Your configuration does not have
```

File: jekyll/_cci2/language-ruby.md (hunk 1 above)
Comment (ganezasan) [reply jekyll/_cci2/language-ruby.md]: The repository is still using the `0.1.2` version, so this is not a necessary change.

**Comment by teesloane** [comment]
Thanks for the review @ganezasan  - we will hold off merging this until the 2.1 guide is merged into master. (and in which case, I will update links in this guide accordingly)

**Pull request merged**